translate/storage/base.py

```diff
@@ -795,13 +792,10 @@ def save(self):
         """Save to the file that data was originally read from, if
         available."""
         fileobj = getattr(self, "fileobj", None)
-        mode = 'w'
-        if self._binary:
-            mode = 'wb'
+        mode = 'wb'
```

Comment:
Also having the mode string in the `open` call (or wherever the mode is used) makes it easier to understand the code.